File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = nums.length;
        int s= 0; 

        if (n == 0) return 0;

        for(int e = 1 ; e < n ; e ++){
           if(nums[e] != nums[s]){
               s++;
               nums[s] = nums[e];

           }
        }
        return s+1 ;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        n = len(nums)
        l = 1 
        r = 1

        while r < n :
            if nums[l-1] != nums[r]:
                nums[l] = nums[r]
                l += 1
                r += 1
            else:
                r += 1    


        return l        